tests/fast-integration/skr-test/README.md

```diff
@@ -0,0 +1,92 @@
+# SKR test
+
+This test covers SKR (SAP Kyma Runtime).
+
+## File Structure consists of the following parts:
+- **provision** folder contains the scripts for provisioning and de-provisioning the SKR cluster using KEB client
+- **oidc** folder contains the OIDC related tests
+- **commerce-mock** folder consists of two scripts:
+  1. **index.js** contains the commerce mock tests suite
+  2. **prep.js** can be called in stand-alone mode for commerce mock resources preparation
+
```

Comment:
I would present the two test modes in the introductory section, to keep the step instructions cleaner:

Edit: We might want to mention the 3rd option, running only the commerce mock test, here as well. In which cases might testers want to use that?

```suggestion

## Usage Modes

You can use the SKR test in two modes - with and without provisioning.

### With provisioning

In this mode, the test executes the following steps:

1. Provision SKR cluster and register compass resources
2. Run OIDC Test
3. Run Commerce Mock Test
4. De-provision the SKR instance and clean the compass resources
  
### Without provisioning

In this mode, the test additionally needs the following environment variables:
- `SKIP_PROVISIONING`, set to `true`
- `INSTANCE_ID` the uuid of the provisioned SKR instance
- `TEST_SUFFIX` determines the compass scenario and the app assigned to it. If not set, it's randomly generated.

   >**IMPORTANT:** The `TEST_SUFFIX must be 4 characters long and is required for every commerce-mock test run. If the compass scenario with the given suffix already exists, the test tries to reuse it.

In this mode, the test executes the following steps:

1. Ensure SKR exists and register compass resources
2. Run OIDC Test
3. Run Commerce Mock Test
4. Clean the compass resources

```

Reply:
I like the idea, see the updated version